index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Scorecard </title>
+    <h1> Bowling Scorecard </h1>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+  </head>
+  <body style="background-image: url('background_wood.png');">
+
+    <form id="round_one"> ROUND ONE
+      <section>
+        <label for="firstBowl"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_one"></h2>
+    </form>
+
+    <form id="round_two"> ROUND TWO
+      <section>
+        <label for="firstBowl2"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl2"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
```

Comment:
repeating ids for form inputs, should work if you change them to the eexpected value in interface.js